src/scrabble.js

```diff
@@ -1,5 +1,126 @@
-function scrabble() {
+const isStringZero = (str) => {
+  if (str.length === 0) {
+    str += '0'
+  }
+  return str
+}
+
+const stringToArray = (word) => {
+  const letters = []
+  word
+    .toLowerCase()
```

Comment:
why are the .lowercase(), .split, .foreach etc in separate lines without repeating 'word' and could they be all in same line

Reply:
They could be in the same line and it would work the same, it simply for formatting to make is somewhat easier to read